Tasks/VsTest/vstest.ts

```diff
@@ -28,6 +29,7 @@ const systemDefaultWorkingDirectory = tl.getVariable('System.DefaultWorkingDirec
 const workingDirectory = systemDefaultWorkingDirectory;
 let testAssemblyFiles = undefined;
 let resultsDirectory = null;
+const taskProps = { areaCode: '', message: '', tracePoint: '0', isUserError: 'false'};
```

Comment:
result

Reply:
properties small letters

Reply:
discuss with @RanjanarMS. i dont like duplicated data

Reply:
done

---
In reply to: [136252471](https://github.com/Microsoft/vsts-tasks/pull/5156#discussion_r136252471) [](ancestors = 136252471)